apps/console/src/features/applications/components/edit-application.tsx

```diff
@@ -236,6 +238,19 @@ export const EditApplication: FunctionComponent<EditApplicationPropsInterface> =
         }
     },[ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template?.id) {    
```

Comment:
```suggestion
        if (template) {    
```

Reply:
Removed this conditional check in 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b because the optional chaining operator was used.